src/SassBuilder.php

```diff
@@ -97,6 +126,17 @@ public static function guessCssNameFromSassFile(string $sassFile, string $output
         return $outputDirectory.'/'.$fileName.'.output.css';
     }
 
+    private function configureOptions(array $options = []): array
+    {
+        $validOptions = array_keys(self::DEFAULT_OPTIONS);
+        $invalidOptions = array_diff(array_keys($options), $validOptions);
+        if (\count($invalidOptions) > 0) {
+            throw new \InvalidArgumentException(sprintf('Invalid Sass options: "%s". Valid options are: "%s".', implode('", "', $invalidOptions), implode('", "', $validOptions)));
+        }
+
+        return array_diff_assoc($options, self::DEFAULT_OPTIONS);
```

Comment:
So here we return values only different from the defaults in `self::DEFAULT_OPTIONS`?